docs/admin/install/serving/install-serving-with-yaml.md

```diff
@@ -194,10 +194,13 @@ Follow the procedure for the DNS of your choice:
     kubectl apply -f {{ artifact(repo="serving",file="serving-default-domain.yaml")}}
     ```
 
-    !!! info "CAVEAT"
-        This will only work if the cluster LoadBalancer service exposes an
+    !!! info "Caveat"
```

Comment:
again, can we make a rule maybe to use warnings instead of "caveats", this seems like overcomplicating it

same comments as above per work list etc for following paragraph

Reply:
Actually, could this be a snippet instead? That way the info doesn't need to be updated in two places now or in the future?

Reply:
We can, but this is the existing text, and Id like to avoid blowing up a small PR to correct some misleading info in to a generic improvement of the document (though Im happy to PR improvements in another issue)

Reply:
It's not a huge change though, it seems like it would be more work to open an issue and raise another PR for this to just change one line...

Reply:
I mean we've literally here started talking about creating a Snippet. Which is fine, but we can iterate on that in a different PR and get it right without tying it in with this correction?